ACKImagePicker/ViewControllers/ACKPhotosViewController.swift

```diff
@@ -241,21 +241,17 @@ extension ACKPhotosViewController: UICollectionViewDataSource {
 
 extension ACKPhotosViewController: UICollectionViewDelegate {
     func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        guard let object = fetchResult?.object(at: indexPath.item) else { assertionFailure(); return }
-
         if let maxNumberOfImages = delegate?.maximumNumberOfSelectedImages, selectedImages.count >= maxNumberOfImages {
             collectionView.deselectItem(at: indexPath, animated: false)
             return
         }
 
-        selectedImages.add(object)
+        selectedImages.append(indexPath.item)
```

Comment:
Do we know that array will work correctly without checking for duplicates?

Reply:
Or you can use https://github.com/apple/swift-collections

Reply:
Yes, but I don't think it is worth adding a dependency..